packages/builder/src/components/backend/DataTable/modals/CreateEditColumn.svelte

```diff
@@ -143,6 +145,10 @@
       )?.[0]
       // Set the tableId based on the selected table
       editableColumn.tableId = relationshipTableIdSecondary
+    } else if (editableColumn.subtype === USER_TYPE) {
+      editableColumn.relationshipType = RelationshipType.ONE_TO_MANY
+    } else if (editableColumn.subtype === USERS_TYPE) {
+      editableColumn.relationshipType = RelationshipType.MANY_TO_MANY
```

Comment:
Can you update the `BBReferenceFieldMetadata` type in `types/src/documents/app/table/schema.ts` to include the `relationshipType` field as the `RelationshipType` enum - just for future reference, while it isn't used on backend, good to know the structures used.